### ISSUE TITLE
mutations?

### ISSUE THREAD

**Issue by PurpleStarCB** [lucasconstantino/graphql-modules#3]
how to add mutations?

**Comment by lucasconstantino** [comment]
Each module object consists of the following:

Key                           | Type     | Description
----------------------------- | -------- | -----------
schema                        | String   | Schema definition string. Should define GraphQL types.
queries                       | String   | Query definition string. Should define root queries.
mutations                     | String   | Mutation definition string. Should define root mutations.
subscriptions                 | String   | Subscription definition string. Should define root subscriptions.
resolvers                     | Object   | Resolver definition object.
resolvers.queries             | Object   | Query resolver map.
resolvers.queries.[key]       | Object   | Query resolver function.
resolvers.mutations           | Object   | Mutation resolver map.
resolvers.mutations.[key]     | Object   | Mutation resolver function.
resolvers.subscriptions       | Object   | Subscription resolver map.
resolvers.subscriptions.[key] | Object   | Subscription resolver function.
modules                       | Array    | An array of submodules/dependencies.
alter                         | Function | A function to alter the resulting configuration object after it's being created.

I'll eventually add this to the documentation. So, basically mutations are very similar to queries.

**Obs.: I created this package out of necessity in the beginning of a project. There are plenty of functionality missing. It still does not solve type field resolvers, for instance. It's source code is very, very simple, though. Feel free to send a pull-request for anything you feel is missing; I hope to add a bunch while my project goes on.**

**Obs.2: Still, I added this `alter` key - which is basically a function to edit the generated schema configuration object. This gives you conditions to virtually add anything missing directly to this configuration object, no needs to way for new features to be supported by `graphql-modules`.**
